pyrgg/pyrgg.py

```diff
@@ -369,9 +374,9 @@ def branch_gen(
             else:
                 used_vertices[random_tail] = [vertex_index]
         if sign == 2:
-            random_weight = random_system.randint(min_weight, max_weight)
+            random_weight = random_unit(min_weight, max_weight)
         else:
-            random_weight = sign_gen() * random_system.randint(min_weight, max_weight)
+            random_weight = sign_gen() * random_unit(min_weight, max_weight)
```

Comment:
Desired probablity distribution can be considered in future too.

Reply:
Good point for future ;-)